cli/src/opts/cast.rs

```diff
@@ -121,6 +121,16 @@ pub enum Subcommands {
         #[clap(long, env = "ETH_RPC_URL")]
         rpc_url: String,
     },
+    #[clap(name = "logs")]
+    #[clap(about = "Print the raw logs generated by an address")]
+    Logs {
+        #[clap(long = "block", short = 'b', parse(try_from_str = parse_block_id) )]
+        block: Option<BlockId>,
+        #[clap(long= "address", parse(try_from_str = parse_name_or_address))]
+        address: NameOrAddress,
+        #[clap(long, env = "ETH_RPC_URL")]
+        rpc_url: String,
```

Comment:
switch order,
- address
- block
- rpc_url

and remove the `long` attribute 

this way we can probably get rid of `--address` and the user can instead do `logs 0xasdasd... 12345`

not 100% sure this will work